src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -520,6 +534,7 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		}
 
 		this._refreshInlineCompletion();
+		this._addPropertiesToInlineCompletionItem(this._model?.items.map(i => i.completion) || []);
```

Comment:
edit: this doesn't work because the item is not removed from the model in the case that I backspace and type again - working on it.

Reply:
fixed